modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPAttachmentFileEntryLocalServiceImpl.java

```diff
@@ -257,8 +260,8 @@ else if (Validator.isNotNull(externalReferenceCode)) {
 					displayDateDay, displayDateYear, displayDateHour,
 					displayDateMinute, expirationDateMonth, expirationDateDay,
 					expirationDateYear, expirationDateHour,
-					expirationDateMinute, neverExpire, titleMap, json, priority,
-					type, serviceContext);
+					expirationDateMinute, neverExpire, true, titleMap, json,
```

Comment:
Why true instead of the new parameter?

Reply:
Overlooked, I'll change it immediately, thanks!